.github/workflows/ci-github-hosted.yml

```
@@ -169,14 +184,21 @@ jobs:
           # The upload step later requires a non-relative path, hence the realpath
           echo "spack-env-dir=$(realpath $SPACK_ROOT/../environments)" >> $GITHUB_OUTPUT
 
+          container_id=${{ job.container.id }}
+          short_container_id=$(echo ${{ job.container.id }} | cut -c1-8)
```

```suggestion
          short_container_id=${container.id:0:8}
```
Can use `bash` syntax to achieve the same effect

e.g.

```bash
$ container_id=2e38fecebe37d32a47efa79f5148f83ef23cf7f48ff7f460fddb27e77d2c03d7
$ echo ${container_id:0:8}
2e38fece
$ echo $(echo 2e38fecebe37d32a47efa79f5148f83ef23cf7f48ff7f460fddb27e77d2c03d7 | cut -c1-8)
2e38fece
```